curvefs/src/metaserver/storage/rocksdb_storage.h

```diff
@@ -35,8 +35,10 @@
 #include "rocksdb/options.h"
 #include "rocksdb/filter_policy.h"
 #include "rocksdb/slice_transform.h"
+#include "rocksdb/table_properties.h"
```

Comment:
it seems you didn't directly use names from this header file in current header file, so put these include in source file in enough.

Reply:
> it seems you didn't directly use names from this header file in current header file, so put these include in source file in enough.

fixed.